signature-aggregator/aggregator/aggregator.go

```diff
@@ -39,10 +39,10 @@ type blsSignatureBuf [bls.SignatureLen]byte
 
 const (
 	// Number of retries to collect signatures from validators
-	maxRelayerQueryAttempts = 5
+	maxRelayerQueryAttempts = 10
 	// Maximum amount of time to spend waiting (in addition to network round trip time per attempt)
 	// during relayer signature query routine
-	signatureRequestRetryWaitPeriodMs = 10_000
+	signatureRequestRetryWaitPeriodMs = 20_000
```

Comment:
Not sure if it's in the description, but why were these values doubled? (Just for my own knowledge)

Reply:
Might not be necessary anymore, but I increased this to account for the increased warm up time when 1) tracking a new subnet or 2) adding/removing validators. With `ValidatorManager` [churn tracking](https://github.com/ava-labs/teleporter/blob/main/contracts/validator-manager/ValidatorManager.sol#L544), the latter case is likely rare, but the former will be common on the signature aggregator side. The current implementation eliminates this warm up latency for the new subnet case, so it's probably safe to reduce this back to the original value.